GitUI/GitModuleForm.cs

```diff
@@ -103,10 +103,15 @@ protected GitModuleForm([NotNull] GitUICommands commands)
 
         protected override bool ExecuteCommand(int command)
         {
-            return ScriptsRunner.RunScript(command, owner: this, UICommands)
+            return ScriptsRunner.RunScript(command, owner: this, UICommands, GetScriptOptionsProvider(null))
                 || base.ExecuteCommand(command);
         }
 
+        public virtual IScriptOptionsProvider? GetScriptOptionsProvider(GitModuleControl source)
```

Comment:
Does it have to be public?

Reply:
Yes, otherwise it can't be accessed from `GitModuleControl`